src/main/java/org/eclipsefoundation/marketplace/service/impl/GuavaCachingService.java

```diff
@@ -47,43 +48,62 @@
 public class GuavaCachingService<T> implements CachingService<T> {
 	private static final Logger LOGGER = LoggerFactory.getLogger(GuavaCachingService.class);
 
-	@ConfigProperty(name = "cache.max", defaultValue = "2500")
+	@ConfigProperty(name = "cache.max", defaultValue = "10000")
 	long maxSize;
-	@ConfigProperty(name = "cache.ttl.access", defaultValue = "21600")
+	@ConfigProperty(name = "cache.ttl.access", defaultValue = "900")
```

Comment:
I like to append the unit for such properties, e.g. `cache.ttl.access.seconds` or `cache.ttl.access.millis` to make it clear to the user without forcing him to read the documentation (which he often does not do ;)). WDYT?

Reply:
another solution is to require a string in a format parsable by [java.time.Duration.parse(CharSequence)](https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#parse-java.lang.CharSequence-) as it includes the time unit in the string. Then you can use it in your `init()` method like this:

    .expireAfterAccess(Duration.parse(ttlAccess))

See https://github.com/eclipse-cbi/macos-notarization-service/blob/master/src/main/java/org/eclipse/cbi/ws/macos/notarization/NotarizationCache.java#L41 for how I use it in another Quarkus app.

Reply:
I ended up removing the access and keeping the `cache.ttl.write` config, updating to be `cache.ttl.write.seconds`